src/main.zig

```diff
@@ -2887,7 +2887,7 @@ fn printErrMsgToFile(
     color: Color,
 ) !void {
     const color_on = switch (color) {
-        .auto => file.isTty(),
+        .auto => file.isTty() and !(file.isDumbTerm()),
```

Comment:
Just a [heads up](https://no-color.org/).

```suggestion
        .auto => file.supportsAnsiEscapeCodes(),
```

Reply:
This wont work because you will want colors on windows, but it supports no ansi escape codes.